Scripts/ParseEmailFiles/ParseEmailFiles.py

```diff
@@ -3536,8 +3536,7 @@ def handle_eml(file_path, b64=False, file_name=None, parse_only_headers=False, m
 
                 else:
                     # .msg and other files (png, jpeg)
-                    if part.is_multipart() and part.get_content_type() == 'message/delivery-status' \
-                            and max_depth - 1 > 0:
+                    if part.is_multipart() and max_depth - 1 > 0:
```

Comment:
what is `message/delivery-status` and why is it not needed?

Reply:
From this source:
https://tools.ietf.org/html/rfc3464#section-2.1
![image](https://user-images.githubusercontent.com/46249224/78112804-6d907c00-7407-11ea-85a7-0056a67c4e3f.png)